db/migrations/20201211090800_create_buildpacks.up.sql

```diff
@@ -0,0 +1,15 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE IF NOT EXISTS buildpacks(
+   id uuid DEFAULT uuid_generate_v4 (),
+   namespace VARCHAR (250) NOT NULL,
+   bp_name VARCHAR (250) NOT NULL,
```

Comment:
@jkutner we could probably just call this `name` (without the `bp` prefix)

Reply:
i thought `name` was a reserved word. But I might have been wrong. in any case, we could quote it.

Reply:
Hmm, It might be.  If so, I'm fine leaving it.

Reply:
fixed in 58a2513